### PR TITLE
kubetest allow extracting from patch version

### PR DESCRIPTION
**Pull request by chaodaiG**
Also allow `-gke` at the end. 

/assign @krzyzacy 
/cc @krzyzacy

### CODE REVIEW

File: kubetest/extract_test.go [hunk 1]
```diff
@@ -137,6 +137,16 @@ func TestExtractStrategies(t *testing.T) {
 			"https://storage.googleapis.com/gke-release-staging/kubernetes/release",
 			"v1.2.3+abcde",
 		},
+		{
+			"ci/gke-latest-1.13.0",
+			"https://storage.googleapis.com/gke-release-staging/kubernetes/release",
+			"v1.2.3+abcde",
+		},
+		{
+			"ci/gke-latest-1.13.0-gke",
```
Comment (krzyzacy) [inline kubetest/extract_test.go]: I obviously didn't write the most useful test

File: kubetest/extract_k8s.go [hunk 2]
```diff
@@ -74,7 +74,7 @@ func (l *extractStrategies) Set(value string) error {
 	var strategies = map[string]extractMode{
 		`^(bazel)$`: localBazel,
 		`^(local)`:  local,
-		`^gke-?(default|channel-(rapid|regular|stable)|latest(-\d+.\d+)?)?$`: gke,
+		`^gke-?(default|channel-(rapid|regular|stable)|latest(-\d+.\d+(.\d(-gke)?)?)?)?$`: gke,
```
Comment (krzyzacy) [inline kubetest/extract_k8s.go]: I think the patch version should also be `\d+`

File: kubetest/extract_test.go (hunk 1 above)
Comment (chaodaiG) [reply kubetest/extract_test.go]: lol

File: kubetest/extract_k8s.go (hunk 2 above)
Comment (chaodaiG) [reply kubetest/extract_k8s.go]: done, and added a test for it

**Pull request merged**